types/chat.ts

```diff
@@ -155,7 +158,10 @@ export interface QuickActions {
 }
 
 export interface TabData {
```

Comment:
Should we add new properties to support setting tab header? (`tabHeaderDetails` property on vscode repo)

Reply:
I see you added `title`, but we still need `description` and header-specific `icon`. Shall we introduce this `tabHeader` as a new type in the protocol?